Sources/MapboxDirections/DirectionsCredentials.swift

```diff
@@ -24,14 +24,21 @@ public struct DirectionsCredentials: Equatable {
      */
     public var skuToken: String? {
         #if !os(Linux)
-        guard let mbx: AnyClass = NSClassFromString("MBXAccounts") else { return nil }
-        guard mbx.responds(to: Selector(("serviceSkuToken"))) else { return nil }
-        guard let serviceSkuToken = mbx.value(forKeyPath: "serviceSkuToken") as? String,
-              let serviceAccessToken = mbx.value(forKeyPath: "serviceAccessToken") as? String,
-              serviceAccessToken == accessToken else {
-                  return nil
-              }
-        return serviceSkuToken
+        guard let mbx: AnyClass = NSClassFromString("MBXAccounts"),
+              mbx.responds(to: Selector(("serviceSkuToken"))),
+              let serviceSkuToken = mbx.value(forKeyPath: "serviceSkuToken") as? String
+        else { return nil }
+
+        if mbx.responds(to: Selector(("serviceAccessToken"))) {
```

Comment:
What's the use-case for having `serviceSkuToken` and not having `serviceAccessToken`?

Reply:
Backward compatibility basically. For somebody who might use this library outside of the SDK. We will have `serviceAccessToken` in SDK for sure.